Formula/plank.rb

```diff
@@ -0,0 +1,34 @@
+class Plank < Formula
+  desc "Framework for generating immutable model objects"
+  homepage "https://pinterest.github.io/plank/"
+  url "https://github.com/pinterest/plank/archive/v1.2.tar.gz"
+  sha256 "9ee8c248d7b4488f7f952b01d913b34cd83bd40111092fd49015d1c63b9e885d"
+  head "https://github.com/pinterest/plank.git"
+
+  depends_on :xcode => ["9.0", :build]
+
+  def install
+    system "make", "archive"
+    bin.install ".build/release/plank"
+  end
+
+  test do
+    system "#{bin}/plank", "--help"
+    (testpath/"pin.json").write <<~EOS
+      {
+        "id": "pin.json",
+        "title": "pin",
+        "description" : "Schema definition of a Pin",
+        "$schema": "http://json-schema.org/schema#",
+        "type": "object",
+        "properties": {
+          "id": { "type": "string" },
+          "link": { "type": "string", "format": "uri"}
+         }
+      }
+    EOS
+    system "#{bin}/plank", "--lang", "objc,flow", "--output_dir", testpath, "pin.json"
+    assert_predicate (testpath/"Pin.h"), :exist?, "[ObjC] Generated file does not exist"
```

Comment:
No need for the parens here. `assert_predicate testpath/"Pin.h", :exist?` etc should work just fine.